### ISSUE TITLE
manual/Highlights22.md

### ISSUE THREAD

**Issue by garbagetown** [playframework-ja/translation-project#752]
- https://github.com/playframework-ja/translation-project/tree/2.2.5/manual/Highlights22.md
- diffs between [2.2.0/manual/Highlights22.md](https://github.com/playframework-ja/translation-project/tree/2.2.0/manual/Highlights22.md) and [2.2.5/manual/Highlights22.md](https://github.com/playframework-ja/translation-project/tree/2.2.5/manual/Highlights22.md).

```
--- /Users/garbagetown/Desktop/2.2.0/manual/Highlights22.md 2013-09-19 10:11:22.000000000 +0900
+++ //Users/garbagetown/Desktop/2.2.5/manual/Highlights22.md    2014-10-06 20:07:20.000000000 +0900
@@ -13,7 +13,7 @@
 '''scala
 def index = Action.async {
   val foo: Future[Foo] = getFoo()
-  foo.map(f => Ok(foo))
+  foo.map(f => Ok(f))
 }
 '''


```

**Comment by garbagetown** [comment]
やります :hand:

**Comment by garbagetown** [comment]
merged at #1043
